finetune/base.py

```diff
@@ -447,35 +447,34 @@ def _cached_inference(self, Xs, predict_keys=None, n_examples=None, update_hook=
 
         return predictions
 
-    def _inference(self, Xs, predict_keys=None, n_examples=None, context=None, update_hook=None):
+    def _inference(self, Xs, predict_keys=None, n_examples=None, context=None, update_hook=None, force_build_lm=None):
         Xs = self.input_pipeline._format_for_inference(Xs)
-
         if self._cached_predict:
             return self._cached_inference(
-                Xs=Xs, predict_keys=predict_keys, n_examples=n_examples, update_hook=update_hook
-            )
+                Xs=Xs, predict_keys=predict_keys, n_examples=n_examples, update_hook=update_hook, force_build_lm=force_build_lm)
         else:
             input_fn = self.input_pipeline.get_predict_input_fn(Xs, context=context)
             estimator, hooks = self.get_estimator(
-                build_explain=PredictMode.EXPLAIN in predict_keys
+                build_explain=PredictMode.EXPLAIN in predict_keys,
+                force_build_lm=force_build_lm
             )
             length = len(Xs) if not callable(Xs) else None
 
             prediction_iterator = estimator.predict(
                 input_fn=input_fn, predict_keys=predict_keys, hooks=hooks
             )
             predictions = ProgressBar(
-                prediction_iterator, 
-                total=n_examples or length, 
-                desc="Inference", 
+                prediction_iterator,
+                total=n_examples or length,
+                desc="Inference",
                 update_hook=update_hook
             )
             try:
                 return [
                     pred[predict_keys[0]] if len(predict_keys) == 1 else pred
-                    for pred in predictions
-                ]
+                    for pred in predictions]
             except ValueError:
+                import traceback; traceback.print_exc()
```

Comment:
Should cut this extra traceback call here -- think it's from when we were debugging

Reply:
removed